contracts/staking/StakingTemplate.sol

```diff
@@ -445,16 +445,13 @@ contract StakingTemplate is Ownable {
 
         // our lastRewardBlock isn't up to date, as the result, the availableRewards isn't
         // the right amount that delegator can award
-        if (currentBlock > lastRewardBlock) {
```

Comment:
Why we need remove this situation? There may have several transactions within one block.

Reply:
Because there are sevrial tranctions in one block. If there has a user called _update method and change lastRewardBlock to currentBlock. And another user call getPendingreward method that he will got the wrong result since the 'if condition' has taked.